builtins/amp-img.js

```diff
@@ -141,6 +145,17 @@ export class AmpImg extends BaseElement {
     }
 
     this.propagateAttributes(ATTRIBUTES_TO_PROPAGATE, this.img_);
+
+    if (this.useNativeSrcset_) {
+      this.propagateAttributes(['src', 'srcset', 'sizes'], this.img_);
+
+      const nativeSizes = this.element.getAttribute('native-sizes');
```

Comment:
On link-rel-preload this is called `imgsizes` because `sizes` means something else there. Lets use the same name.